### ISSUE TITLE
Removing existing chunks from region throws server errors when loaded.

### ISSUE THREAD

**Issue by ZombieStriker** [GameModsBR/Region-Manipulator#5]
After removing a bunch of chunks from a region folder, I receive errors like this the next time the server starts up:

>[01:43:06] [chunk IO worker/WARN]: Chunk [-8, -21] is allocated, but stream is missing
[01:43:06] [chunk IO worker/WARN]: Chunk [-7, -21] is allocated, but stream is missing
[01:43:06] [chunk IO worker/WARN]: Chunk [-6, -21] is allocated, but stream is missing
[01:44:38] [chunk IO worker/ERROR]: Chunk [-11, -26] header is truncated: expected 8192 but read 0
[01:44:38] [Server thread/ERROR]: Chunk file at [-9, -26] is in the wrong location; relocating. (Expected [-9, -26], got [-3, -6])
[01:44:38] [Server thread/WARN]: Found invalid structure reference [ Mineshaft @ [2, -4] ] for chunk [-9, -26].
[01:44:38] [chunk IO worker/WARN]: Chunk [-8, -26] is allocated, but stream is missing
[01:44:38] [chunk IO worker/WARN]: Chunk has both internal and external streams
[01:44:38] [chunk IO worker/ERROR]: External chunk path .\world\region\c.-5.-26.mcc is not file
[01:44:38] [chunk IO worker/ERROR]: Chunk [-4, -26] header is truncated: expected 8192 but read 0
[01:44:38] [chunk IO worker/WARN]: Chunk [-3, -26] is allocated, but stream is missing
`

Basically, you should be able to remove almost all of a chunks in a region to recreate the issue. Here's a copy of the region files that gets corrupted:
[r.-1.-1.zip](https://github.com/GameModsBR/Region-Manipulator/files/4602775/r.-1.-1.zip)
